build/release-tars/BUILD.bazel

```diff
@@ -26,7 +26,7 @@ release_filegroup(
             ":cert-manager-manifests.tar.gz",
         ],
         for_client = [
-            ":cert-manager-ctl-{OS}-{ARCH}.tar.gz",
+            ":krew-kubectl-cert-manager-{OS}-{ARCH}.tar.gz",
```

Comment:
I'd expect this to still be prefixed `cert-manager` similar to the other archives in this group (so `:cert-manager-kubectl-cert_manager.tar.gz`?)

Reply:
just noticed that!